holoviews/ipython/__init__.py

```diff
@@ -179,18 +179,21 @@ def __call__(self, *args, **params):
                                                                               fig='svg')
                         holoviews.archive.exporters = [svg_exporter] +\
                                                       holoviews.archive.exporters
-                OutputMagic.allowed['backend'] = list_backends()
-                OutputMagic.allowed['fig'] = list_formats('fig', backend)
-                OutputMagic.allowed['holomap'] = list_formats('holomap', backend)
+
+                Store.output_options = OutputOptions()
```

Comment:
When would a user every look at the ``OutputOptions``? Seems like internal API to me.